### PR TITLE
Update Duktape to 1.3.0

### PR DESCRIPTION
**Pull request by MichaelEvans**

### CODE REVIEW

**Comment by MichaelEvans** [comment]
I guess this isn't as straightforward as I had hoped - based on the modifications in #17

**Comment by swankjesse** [comment]
Next steps . . . we need to keep timezones working. Want to fix the C code? Or back this out?

**Comment by swankjesse** [comment]
Once we have the timezone stuff worked out, this change will make more sense.

**Comment by MichaelEvans** [comment]
I updated the script in #26 to patch `duk_config.h` with the necessary changes in order to keep the timezone bits working. This PR was generated from that script (I'm hardly a bash wizard, that script can probably be improved) but it maintains the timezone tweaks that were in place.

File: duktape/src/main/jni/duk_custom.h [hunk 1]
```diff
@@ -0,0 +1 @@
+#define DUK_USE_DATE_GET_LOCAL_TZOFFSET android__get_local_tzoffset
```
Comment (JakeWharton) [inline duktape/src/main/jni/duk_custom.h]: Trailing newline

/me ducks

Comment (MichaelEvans) [inline duktape/src/main/jni/duk_custom.h]: don't you mean `/me duks`

Comment (JakeWharton) [inline duktape/src/main/jni/duk_custom.h]: touché

**Comment by swankjesse** [comment]
This is awesome.

**Pull request merged**